VideoLocker/src/main/java/org/edx/mobile/module/notification/NotificationPreference.java

```diff
@@ -0,0 +1,85 @@
+package org.edx.mobile.module.notification;
+
+import org.edx.mobile.model.api.CourseEntry;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * We extends HashMap to make the implementation extendible
+ */
+public class NotificationPreference extends ArrayList<EdxLocalParseChannel> implements Serializable{
```

Comment:
@aleffert for most domain objects, i used to add seriazable interface and it is required by some frameworks.   it does not harm here as we do save  it as a json string, not a java object.